stonesoup/tests/test_stitcher.py

```diff
@@ -0,0 +1,156 @@
+from datetime import datetime, timedelta
+import numpy as np
+
+from stonesoup.types.state import GaussianState
+from stonesoup.models.transition.linear import CombinedLinearGaussianTransitionModel, \
+    ConstantVelocity, OrnsteinUhlenbeck
+from stonesoup.predictor.kalman import KalmanPredictor
+from stonesoup.updater.kalman import KalmanUpdater
+from stonesoup.hypothesiser.distance import DistanceHypothesiser
+from stonesoup.measures import Mahalanobis
+from stonesoup.types.detection import TrueDetection
+from stonesoup.models.measurement.linear import LinearGaussian
+from stonesoup.dataassociator.neighbour import GNNWith2DAssignment
+from stonesoup.deleter.error import CovarianceBasedDeleter
+from stonesoup.deleter.multi import CompositeDeleter
+from stonesoup.deleter.time import UpdateTimeStepsDeleter
+from stonesoup.initiator.simple import MultiMeasurementInitiator
+from stonesoup.types.groundtruth import GroundTruthPath, GroundTruthState
+from stonesoup.stitcher import TrackStitcher
+
+start_time = datetime.now().replace(second=0, microsecond=0)
+np.random.seed(100)
```

Comment:
Can't really use seed in test (without resetting it again). Probably best to use seed in models, and also capture example starting points from this seed for now.